Update view-user-accounts-with-microsoft-365-powershell.md (#2961)
MicrosoftDocs/microsoft-365-docs: microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell.md

Get-AzureADUser -ObjectID <sign-in name of the user account> | Select DisplayName,UserPrincipalName,AccountEnabled
```

> ### View account Synchronization Status

Reviewer: ```suggestion
### View account synchronization status
```

> The User accounts have two sources, Windows Server AD which are  accounts that sync from on-prem AD to  cloud and Azure Active Directory which are accounts directly created in cloud.

Reviewer: ```suggestion
User accounts have two sources; Windows Server Active Directory (AD) which are accounts that sync from on-premises AD to the cloud and Azure AD which are accounts directly created in the cloud.
```

```powershell
Get-AzureADUser | Where {$_.DirSyncEnabled -eq $true}
```

> This command instructs PowerShell to get all users who have the attributte Dirsyncenabled set to True. This can be used to pull up accounts synchronozing from on premise AD.

Reviewer: ```suggestion
This command instructs PowerShell to get all users who have the attribute Dirsyncenabled set to True. It can be used to pull up accounts synchronizing from on-premise AD.
```

> This command instructs PowerShell to get all users who have the attribute Dirsyncenabled set to True. It can be used to pull up accounts synchronizing from on-premise AD.

Reviewer: ```suggestion
This command instructs PowerShell to get all users who have the attribute **DirSyncEnabled** set to True. It can be used to pull up accounts synchronizing from on-premise AD.
```

```powershell
Get-AzureADUser | Where {$_.DirSyncEnabled -ne $true}
```

> This command instructs PowerShell to get all users who have the attributte Dirsyncenabled set to False. This can be used to pull up Cloud only accounts.

Reviewer: ```suggestion
This command instructs PowerShell to get all users who have the attribute Dirsyncenabled set to False. It can be used to pull up cloud-only accounts.
```

> This command instructs PowerShell to get all users who have the attribute Dirsyncenabled set to False. It can be used to pull up cloud-only accounts.

Reviewer: ```suggestion
This command instructs PowerShell to get all users who have the attribute **DirSyncEnabled** set to False. It can be used to pull up cloud-only accounts.
```
